RecoLocalCalo/HcalRecProducers/src/HBHEPhase1Reconstructor.cc

```diff
@@ -483,7 +485,8 @@ void HBHEPhase1Reconstructor::processData(const Collection& coll,
             auto s(frame[ts]);
             const uint8_t adc = s.adc();
             const int capid = s.capid();
-            const double pedestal = calib.pedestal(capid);
+            //optionally increase pedestal mean to account for dark current + crosstalk
+            const double pedestal = calib.pedestal(capid) + (saveEffectivePedestal_ ? darkCurrent * 25. / (1. - lambda) : 0.);
```

Comment:
what's saved in that case is ``` calib.pedestal(capid) + effective ```, so, it's a sum, not the actual values.
Better have a less confusing name.